packages/tool-cache/src/manifest.ts

```diff
@@ -92,6 +92,11 @@ export async function _findMatch(
             chk = true
           } else {
             chk = semver.satisfies(osVersion, item.platform_version)
+            if (!chk) {
+              debug(
+                `os version: "${osVersion}" does not match version python is built for: "${item.platform_version}"`
```

Comment:
You can put capital 'OS' instead of lowercase 'os'

Reply:
Done